vlsi/Makefile

```diff
@@ -0,0 +1,122 @@
+#########################################################################################
+# vlsi makefile
+#########################################################################################
+
+#########################################################################################
+# general path variables
+#########################################################################################
+base_dir=$(abspath ..)
+vlsi_dir=$(abspath .)
+sim_dir=$(abspath .)
+
+#########################################################################################
+# include shared variables
+#########################################################################################
+include $(base_dir)/variables.mk
+
+#########################################################################################
+# import other necessary rules and variables
+#########################################################################################
+include $(base_dir)/common.mk
+
+#########################################################################################
+# vlsi types and rules
+#########################################################################################
+
+#sim_name is unused, but GenerateSimFiles expects it
+sim_name     ?= vcs
```

Comment:
What is this set for?

Reply:
https://github.com/ucb-bar/project-template/blob/9622a9757067e9c1dab9a4baf551ce932dc87b54/common.mk#L34-L35